src/PowerDynamics.jl

```diff
@@ -48,6 +48,9 @@ include("lines/RLLine.jl")
 include("operationpoint/operationpoint.jl")
 include("operationpoint/find_valid_initial_condition.jl")
 
+include("simulations/AbstractNodePerturbation.jl")
```

Comment:
I checked out this PR but this file seems to be missing.

Reply:
Sry, I added some files :D that were missing